seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-oracle-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/oracle/OracleCDCIT.java

```diff
@@ -65,6 +67,7 @@
         type = {EngineType.SPARK, EngineType.FLINK},
         disabledReason =
                 "Currently SPARK do not support cdc,Flink is prone to time out, temporarily disable")
+@TestMethodOrder(MethodOrderer.MethodName.class)
```

Comment:
why add this？

Reply:
Running in parallel may cause insufficient JDBC connections

Reply:
https://junit.org/junit5/docs/5.5.0/api/org/junit/jupiter/api/TestMethodOrder.html
https://junit.org/junit5/docs/current/user-guide/#writing-tests-parallel-execution

By default, JUnit Jupiter tests are run sequentially in a single thread.

Reply:
So I'll delete this and test it again